java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk

```diff
@@ -76,7 +76,7 @@ ifeq ($(TFLITE_ENABLE_XNNPACK_DELEGATE),true)
 TFLITE_FLAGS += -DTFLITE_XNNPACK_DELEGATE_SUPPORTED=1
 endif
 
-TF_LITE_DIR := $(LOCAL_PATH)/tensorflow-lite
+TF_LITE_DIR := $(TFLITE_ROOT)
```

Comment:
Can you ensure that ```TFLITE_ROOT``` is already defined when you reach here?

Reply:
```
ifeq ($(ENABLE_TF_LITE),true)
NNS_API_FLAGS += -DENABLE_TENSORFLOW_LITE=1
NNS_SUBPLUGINS += tensorflow-lite-subplugin
include $(LOCAL_PATH)/Android-tensorflow-lite.mk
endif
```

This file in included only if `ENABLE_TF_LITE` is true